extensions/mssql/src/hdfs/webhdfs.ts

```diff
@@ -401,26 +415,6 @@ export class WebHDFS {
 		});
 	}
 
-	/**
-	 * Get file status for given path
-	 * @returns void
-	 */
-	public stat(path: string, callback: (error: HdfsError, fileStatus: any) => void): void {
-		this.checkArgDefined('path', path);
-
-		let endpoint = this.getOperationEndpoint('getfilestatus', path);
```

Comment:
I am confused - why do we no longer support stat, and why do we need to use liststatus instead for exists? 
- I know we might not use this externally but this is helpful when you explicitly want to just list individual file status?
- Isn't there an opportunity to put "convert JSON FileStatus to FileStatus object" in common code and share here to do what you're wanting to do?

Reply:
It's not that we don't support it - it's just that at the webhdfs level there's no "stat" operation so it's confusing to me to look at this and try to figure out the mapping between the IFileSource name and the WebHDFS name.

That being said - after changing this I looked at the others done in the same way I decided that it wasn't a big enough issue (and would just require extra effort without any real gain) and so don't think the rest need to be updated So if you want I can revert this - but I figure that having this be updated isn't going to hurt anything and didn't want to spend more time undoing the change (since I still modified what it returns to be more complete).

Reply:
As to your other points:
- listStatus is for individual files (directories too), so not sure what you're referring to
- Yeah that might be useful, I just didn't want to spend the time on it since it'd just be a one-off thing. If you think it's worth the effort I'll file an issue and add it to the backlog but it really wouldn't be getting us a lot of benefits that I can see